.github/workflows/lint.yml

```diff
@@ -17,8 +17,8 @@ jobs:
       - uses: Swatinem/rust-cache@v2
       - run: cargo fmt --check -- --color=always
       - run: cargo fmt --check --manifest-path fuzz/Cargo.toml
-      - run: cargo clippy --color=always -- -D warnings -A clippy::manual-non-exhaustive
-      - run: cargo clippy --color=always --target x86_64-pc-windows-msvc -- -D warnings -A clippy::manual-non-exhaustive
+      - run: cargo clippy --color=always -- -D warnings
```

Comment:
Would prefer to move this commit before the commit that duplicates this stuff for Windows.